salt/_modules/metalk8s_drain.py

```diff
@@ -56,7 +56,7 @@ def __str__(self):
 
 class DrainTimeoutException(DrainException):
     '''Timeout-specific drain exception.'''
-    pass
+    pass  # pylint: disable=unnecessary-pass
```

Comment:
Can't it be removed instead?

Reply:
It could, but maybe we want to keep this one for code clarity, no strong opinion on this one, with a `pass` or without both ok for me

Reply:
From what I've seen in many Python codebases, the `pass` is usually omitted when possible (like in this case, because there's a docstring).
Exception subclasses are often using this approach: see https://github.com/saltstack/salt/blob/master/salt/exceptions.py#L72-L117 for instance.
So IMHO, we should remove it and let pylint check that we remain consistent.

Reply:
Ok fair I remove this one